### ISSUE TITLE
Remove Borers

### ISSUE THREAD

**Issue by nuklearcellphoneg** [d3athrow/vgstation13#13800]
Yeah, they're fucked, just remove them. The new features have piled upon each other to make some trash balance.

- Xray in five minutes
- Head borers have far too many options, see picture below
- Too many borers in one body
- Xray in five minutes...
- The borers locations do not correlate with the abilities to get very well
- If you stack everything you possibly can stack from a head borer, you're healing 3 burn damage every 2 seconds1 brute damage every 2 seconds, 1 toxin damage every 2 seconds, infinite suffocation damage every 2 seconds(not entirely relevant, but you can survive being critted and left to die)
- You're faster than regular speed even with magboots on
- You're immune to atmospheric temperatures
- Borer speak is a relic left over from it being an antag- there is NO good reason for them to have that any more.
pssst they were fine before but some recent changes have been lacking forethought

**Comment by SonixApache** [comment]
buffslugs

**Comment by ghost** [comment]
just obliterate head borers

**Comment by 9600bauds** [comment]
Please make this into an actual discussion, say why are they fucked and such

**Comment by ThePlapatin** [comment]
"balance"

yeah dude this is a fucking fighting game, gotta keep it balanced

**Comment by nuklearcellphoneg** [comment]
I agree!

**Comment by Rattlederp** [comment]
The moment we start talking in detail about balancing is the moment we turn into tg, let's not. They had to make their plasmamen shitty because people cried about muh balance. This game is years out of date and the combat system is probably the worst part, why even bother

**Comment by ThePlapatin** [comment]
I actually do think there's some point to this pr though, let's say I am an antag and I kill my target and hide the body. My target had a borer I had no way of knowing about. The borer tells his borer friends that im a bad guy and where the target's body is hidden. I get valided and my target is cloned, there was no way of me knowing he had a borer and it's impossible to stop.

On the opposite hand, borers are used rarely so it's not a problem and the game isn't about winning as an antag.

**Comment by ghost** [comment]
i agree that borers are fucked. my opinion is that head borer is just way too fucking powerful. head borer needs to be completely gutted. 
![image](https://cloud.githubusercontent.com/assets/24782063/22611135/f1777a5e-ea36-11e6-89ec-1f7860e0961e.png)

the ones i crossed out make almost no sense to be in the head, some should be moved to chest or just removed in general

**Comment by Exxion** [comment]
The reason they were in the head is because borers could originally only go in the head, and those abilities are from that time

**Comment by AutismSpider** [comment]
Most of the chems head borer has should be chest rezadone peridaxon  charcoal burn/brute heal and Leporazine/dex take out useless chilly juice they get and heat and burn res for that.

leg borers get hyperzine.

Arms are fine you give up chems for a tool that can be use for combat  or to tank damage/block/dodge.

head gets to keep  inaprovaline rye take out space drugs for Methylin make it cost 100 points to unlock use of it.

**Comment by Killette2** [comment]
I suggested that head borers shouldn't have the liver function boost and that should be torso, but was greeted with "lul thas retarded"...

**Comment by Exxion** [comment]
Who the fuck said that? As far as I'm aware, the only reason liver boost is still with the head rather than the torso is because nobody's bothered to move it.

**Comment by Killette2** [comment]
Never mind a different issue of mine was the retarded one, the liver one was just ignored.

https://github.com/d3athrow/vgstation13/issues/11364

**Comment by Lokiamis** [comment]
The other day I had x-ray vision and completely negated a lethal injection syringe with just one borer. They were interesting when they fought for control of your body and reproduced wildly, but now they're just super-PAIs, and I don't think anybody would be happy with what they're capable of if their abilities were given to PAIs in the first place.

**Comment by Lokiamis** [comment]
They're also a way for metafriends to buff the ever living fuck out of each other in a completely ban-proof way.

**Comment by Killette2** [comment]
I'm sure if a guy kept going into xenobio and the same guy kept becoming his borer, that might raise a red flag.

**Comment by Exxion** [comment]
I don't think that's actually against the rules, unless they're also metacommunicating

**Comment by Aurx** [comment]
Problems with borers (some of these apply to pAIs too):

Problem: There's no way to know John Target has a borer until he gets back up and murders you.  
Fix: Make borers obvious.  I'd suggest periodic flavor messages about something squirming in their LOCATION, and flavor messages when they get attacked.  

Problem: Knowing John Target has a borer doesn't mean you can do jackshit about it.  
Fix: Borers should take damage when their body part gets attacked and when locationless damage is taken.  

Problem: Borers do asstons of things.  Borers stack.  
Fix: Pare down the borer abilities.  Borers should be only able to affect their current location.  Borers should be able to move around in their host's body.  Retain one borer to a zone.  Limit how many things a borer can do at once.  

tl;dr Borers should be obvious living bandaids that you can kill at the same time as their host, not invisible invincible living medbays that give super powers.

**Comment by Killette2** [comment]
Head borers make your head swell really huge like the "too smart for" meme and disable headgear

**Comment by Aurx** [comment]
>I don't think that's actually against the rules, unless they're also metacommunicating

It's a tad suspicious, but "Head to xenobio asap to get borer" and "Become borer asap" are both reasonable things players could do independently of each other.  
Easy to test for, though.  Swipe the borer slot and see how the xenobio rusher reacts, and have somebody get to xenobio first and see what the borer rusher does.

**Comment by Killette2** [comment]
Mybe cause bone breaking to harm the borer?
When a borer is secreting chemicals into a limb make the limb flash a green glow while the chemicals are being metabolized

**Comment by AutismSpider** [comment]
would it be silly make the limb that has the borer to pulse or for the limb to look green or swell up+look bit green or red?

@Killette2 salt is deadly to them and Silicate stops the use of most of the bone powers not sure if the sugar thing was taken out or if it works for that matter.

**Comment by Shadowmech88** [comment]
We were discussing head borer tweaks in the Discord some time ago, I just never got around to implementing them.
For example, instead of allowing head borers to give their humans X-ray vision, allow head borers to channel chems to temporarily give themselves X-ray vision.
I think there's definitely more interesting things that can be done with head borers than just leaving them as chem dispensers.

**Comment by Aurx** [comment]
@AutismSpider 
OK, but how do you know to apply the salt/silicate to John Target before robusting him?

**Comment by kol1th** [comment]
They don't add anything to the game, really. I am literally only killed by borers poisoning me with space drugs and/or radium when I get one.

**Comment by ThePlapatin** [comment]
YOU don't add anything to the game

haha got eeem

**Comment by nuklearcellphoneg** [comment]
Just get rid of the x-ray vision entirely, in my opinion... that is far far far too overpowered.

**Comment by ghost** [comment]
should wider vision stay?

**Comment by nuklearcellphoneg** [comment]
Personally... that is a 

HUGE

advantage.

Personally, I don't believe so.

**Comment by Aurx** [comment]
Basically everything borers currently do should go.

**Comment by nuklearcellphoneg** [comment]
I've been adding all the points brought up as they are brought up, in the original post. :^)

**Comment by AutismSpider** [comment]
@Aurx asking add in a pulse or mark to limb to show they have a borer. XXX right arm looks bit lumpy and bit green etc

**Comment by 9600bauds** [comment]
Let's make a list of things everyone can agree on

Can everyone agree that xray should go?
Can everyone agree that some/most headborer chemicals should be moved to chestborers?

**Comment by Shadowmech88** [comment]
I'm fine with hosts not getting xray.
I think it'd be neat for it to be a costly channeling ability for head borers themselves, though.

I think it'd be fine to move some headborer chems to the chest.

**Comment by Kammerjunk** [comment]
>If you stack everything you possibly can stack from a head borer, you're healing 3 burn damage every 2 seconds1 brute damage every 2 seconds, 1 toxin damage every 2 seconds, infinite suffocation damage every 2 seconds(not entirely relevant, but you can survive being critted and left to die)

This is not and was not meant to be an argument againt borers. Asides from a few things (x-ray, hyperzine, and to a limited extent anti-tox come to mind), the borer will not help you in a fight to the death (as opposed to a fight to the crit). It's also slightly wrong, it's 2.5 burn, not 3, which means that each laser shot takes takes 24 seconds to heal.
What I'm saying is that you'll want to fix chemistry, not borers (ODs + 1000u limit + ease of creation).

X-ray is really powerful though

**Comment by AutismSpider** [comment]
@Kammerjunk move most of the chems head borer has to the chest and legs make X ray a toggle or take it out having the power to see past walls and  wider vision is bit much one or the other.

Head borer should have rye and pain killers for its chems over all chest should get most of the healing chems leg for any thing that speeds you up  arms are fine.

**Comment by NigglyWiggly** [comment]
Multiborers removed fucking when

having _Cortical_ borers that can go into your foot and then talk into your brain and shit is just silly, and having people just turn into these bio-weapon tier freaks with a sword for one arm and stretch armstrong for the other is dumb and would probably fit changelings better anyway

**Comment by AI-Is-Rogue** [comment]
@NigglyWiggly The cortical part was removed from their name and the voice when they talk comes from the body part they are in, not the head.

**Comment by Kurfursten** [comment]
Don't get rid of them, but dial them back to be more reasonable and come in less archetypes.

Extendoarm is crazy useful, useful enough that if a borer could ONLY use extendo arm it would still be fun to play and extremely powerful. This ability is literally a get out of death free card for any non-antag (get dragged to medbay) or antag who can regen.
EMP borer can be fun too and the EMP is very reasonably balanced (it takes a staggering 100 chems for a small EMP).
Personally, I don't even see anything wrong with borerspeak as a nice little bonus, it makes them interesting. If anything, I'd like to see unhosted borers able to use it.

Do away with the kitchen sink of borers secreting every chemical in the world from hamserum to paismoke. No more xray or telescopic vision.

**Comment by AutismSpider** [comment]
@Kurfursten    hamserum is whats use to make the emp unless you make emp a verb?

only thing that is a kitchen sink is head borer they OD you on bicaridine to stop bleeding and deal with the toxinson top of giving you power to space walk out of perm.

**Comment by Kurfursten** [comment]
@AutismSpider I was wrong. I was looking at the original PR where it claimed it uses Hamserum but he changed it to use a verb at some point.

Here are some real things that borers can synthesize that are a little silly though:
>Latte (the coffee)
>Bustanut
>Frost oil
>pAI smoke
>Radium
>Both tramadol AND paracetamol (isn't one painkiller enough?)
>Iron, blood, dex+, and clotting agent (what's the point in any of the others if you can make dex+?)

This isn't even talking necessarily about balance and how enough chems make you invincible, just out outrageous the list is at times.

**Comment by AI-Is-Rogue** [comment]
Radium for curing viruses, frost oil for lowering the host's heat along with capsaicin for raising it,  and clotting agent stops internal bleeding.

**Comment by AutismSpider** [comment]
Thing is head borer is a walking medbay only thing they cant fix is busted bones.

Chest gets pai smoke and clotting agent and both oil types.Just take out the smoke and the oil and the damage res and move  the healing chems to this borer sub type.

take out bustanuts form leg borer for hyperzine.

@AI-is-rogue Head does better job at this making chest borer frost oil and capsacin useless Chest is what gets radium but no anti toxin.

**Comment by AI-Is-Rogue** [comment]
@AutismSpider Chest gets arithrazine for fixing toxin damage and yeah, the oils aren't great but the temperature resistance genes are wonderful and even better than leporazine, though you can only pick one due to the fact that having one of the genes makes the activation chance for the other so low

**Comment by AutismSpider** [comment]
over all Head borers are over loaded Chest need lot of work done to it it and it has mix of powers that dont fit it on top of being only borer sub type that has power to do full body body scan on the host.

**Comment by N3X15** [comment]
I have plans to nerf the shit out of them so they don't make you a
living God.

Once they're more concrete, I will share.

**Comment by Ephx1** [comment]
Remove all the idiotic borers in other parts of the body, nerf headborers.
Done.

**Comment by PJB3005** [comment]
pAI borers were a mistake.

Borev when.

**Comment by Ephx1** [comment]
Actually, I would be in favor of body part borers had their own breeding system similar to slimes, but only if the resulting offspring could _only_ go in its designated spot, so that it would take lots of effort to become a slug party bus that shrugs off everything.

**Comment by Praecordia** [comment]
make one in every ~10 borers spawn as a classic bodysnatcher borer
suddenly you can't trust your walking chem dispenser egg anymore

**Comment by Ephx1** [comment]
There's little point to being a bodysnatcher borer though, the original mechanic (or how it was supposed to work) gave them constant brain damage at the control of the borer. Not only would you be piloting a retard, but there's no real reason to be piloting the retard as the original borers objectives were just "breed", which now requires you to leave the body.

**Comment by gbasood** [comment]
so remove brain damage and find a way to make bodysnatching borers not shit
I know it's fucking CRAZY but you can completely rework a gamemode if it's shit

**Comment by Ephx1** [comment]
Then why do we have rev?

**Comment by yclatious** [comment]
More importantly, who do we still have resquad? Rev might not be the best, but squad is worse.

**Comment by gbasood** [comment]
@Ephx1 code it @yclatious also code it
although I'd prefer people get them removed from rotation instead of removed from code

**Comment by yclatious** [comment]
I can barely copy paste code crate and you want me to remake revsquad.

Nisu desu ne.

**Comment by gbasood** [comment]
I didn't say remake

**Comment by yclatious** [comment]
Then what, you want me to make them all spawn with a sketchkin or something? A IED or something? If you want people to improve the mode, ask the dude who made them to do so.

**Comment by gbasood** [comment]
That's me
Feature request shit you want in it
Or get someone to make a server poll to remove it
Also the literal "IED" ieds are shit, they used to have them and they do fucking nothing

**Comment by yclatious** [comment]
Alright

Get the leader to spawn with a smg and a emag, everyone else with a sketchkin and a ied with glass shards in them for extra anarchy, maybe have it so you get a uplink through pda or something after killing a head, granting you tcs per kill so you can get revolution supplies, a agent id, a limited use flaher, you don't want rev that lasts forever and you don't want revsquad to do that either, actually supply the revs decently and maybe revsquad will become a tolerable or even better mode.

**Comment by gbasood** [comment]
MAKE
IT
ITS
OWN
ISSUE
FUCK
this issue discussion is already derailed enough

**Comment by yclatious** [comment]
¯\_(ツ)_/¯

You do a feedback issue, ill post it, makes no sense for anyone but the creator to make a issue regarding the modification of a gamemode they made.

**Comment by Ephx1** [comment]
Maybe, gut borers, expand upon the pAI range.
Someone on bay was working on making your limbs pAIs, so that could be an extension of robotics?
We could shift some borer abilities that way.

**Comment by AutismSpider** [comment]
@Ephx1 the pai going into hard suits lets the pai take over if some thing happens to the master and so on.

**Comment by AutismSpider** [comment]
@N3X15 take out most of the healing chems head one has with just a head borer you turn into unkill-able fuck head

**Comment by Ephx1** [comment]
No, this is something different that I guess was never expanded upon, I read it while browsing Bay's idea's corner. There was an actual coder behind it, so I expected at least something was done. It was the idea of incorporating pAIs into prosthetic limbs, with a similar concept of "speaking into" the person themself.

You could probably make that into the stupid hookshot, shield, sword thing we have now for borers. It would take more than some bored player to wriggle into you.

**Comment by AutismSpider** [comment]
normal prosthetic limbs or are they harden in some way?

**Comment by Ephx1** [comment]
I don't think it ever got past "let's put pAIs in limbs" but I would assume so, at least a small step up from a normal limb,

**Comment by AutismSpider** [comment]
Limb should be  harden a bit some emp res and bit more hp on it and the build cost plasma, glass, iron,  diamond or uranium

**Comment by gbasood** [comment]
@yclatious no that's dumb
It's a game feature, you can make an issue about it like everyone else constantly has about every other gamemode.
GitHub isn't a forum, you don't need to keep everything about one subject in one issue

**Comment by yclatious** [comment]
You really want me to go ahead and create a issue regarding the discussion of revsquad, start spitballing on how to improve revsquad, which is universally known to be the worst gamemode we currently have and that has almost always ended badly in almost every round it has even been in, when I myself don't even think I played as the initial revsquad ever, have barely interacted with it, only know they start off underequiped for their job and that they have a headset so they can communicate decently/identify themselves?

You of all people should be the one who should make something to fix/improve what you made and get feedback, not some random who that barely, if at all, played as the participant of your gamemode. What's dumb is that you know its in a bad spot and haven't made a issue to discuss it at all, its your job or someone elses, not mine since Im so inexperienced about it, I don't go and talk about massive mechanical fixes about shit I don't know, all I know is that they are undertooled and fail constantly, most I can do is tell you to equip them decently instead of whatever they currently have.

**Comment by nuklearcellphoneg** [comment]
Er

C'mon now. You might see my issues as shitposting or something(Remove borers instead of 'hay guys borers are kind of messed up!), but the intention is to start a global discussion in the community. Let's not derail this conversation, first off, and second, if you want to discuss that game mode and have someone put in a change you suggest, you need to gauge the people and garner support for it. I admin for free, coders code for free.

**Comment by nuklearcellphoneg** [comment]
ie go make an issue, no balls

**Comment by mrpain666** [comment]
I DED PLS NERF

**Comment by AutismSpider** [comment]
@nuklearcellphoneg Borers are being work on #13831

**Comment by nuklearcellphoneg** [comment]
yatta
